Automattic/wp-calypso
# Hosting Metrics: Clean up old Site Logs code

Fixes https://github.com/Automattic/dotcom-forge/issues/3393

## Proposed Changes

* Removes legacy Site logs code.
* Redirects `/site-logs*` to `/site-monitoring*`
* Updates 'Site logs' to 'Site monitoring' in the ellipsis menu.

<img width="306" alt="CleanShot 2023-08-28 at 15 48 09@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/68bc953b-ad0f-4c14-a835-9ef7095f2e4a">


## Testing Instructions

1. Navigate to `/site-logs` and verify you're redirected to `/site-monitoring` as expected.
2. Navigate to `/site-logs/<site>` and verify you're redirected to `/site-monitoring/<site>` as expected.
3. Verify the 'Site monitoring' ellipsis menu item appears and works as expected.
